### PR TITLE
Fix: Use camelCase for fields and variables

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] uses `camelCase` for fields and variables

### CODE REVIEW

File: classes/Http/Form/Form.php [hunk 1]
```diff
@@ -13,12 +13,12 @@
...
-    protected $_cleanData;
-    protected $_taintedData;
-    protected $_purifier;
-    protected $_fieldList = [];
+    protected $options;
+    protected $messages;
+    protected $cleanData;
+    protected $taintedData;
+    protected $purifier;
+    protected $fieldList = [];
```
Comment (localheinz) [inline classes/Http/Form/Form.php]: These changes are potential BC breaks, if anyone else extends `Form`

Comment (mdwheele) [reply classes/Http/Form/Form.php]: I think that's fine and arguably even a non-issue.

File: classes/Http/Form/Form.php [hunk 2]
```diff
@@ -167,8 +167,8 @@ public function hasErrors(): bool
      */
     protected function _addErrorMessage($message)
```
Comment (mdwheele) [inline classes/Http/Form/Form.php]: Should we drop underscore here?

File: classes/Domain/Services/ResetEmailer.php [hunk 3]
```diff
@@ -82,8 +82,8 @@ private function parameters($userId, $resetCode)
                 ? 'https' : 'http',
             'host'    => !empty($_SERVER['HTTP_HOST']) ? $_SERVER['HTTP_HOST'] : 'localhost',
```
Comment (mdwheele) [inline classes/Domain/Services/ResetEmailer.php]: `phpmd` complains about use of `$_SERVER`. We *could* inject an instance of `Request` here but... I don't know that I care. What do you think? And if we don't care, do we want to create an issue to adjust `phpmd` to slack-off?

File: classes/Http/Form/SignupForm.php [hunk 4]
```diff
@@ -43,44 +43,44 @@ class SignupForm extends Form
...
+        $ValidSpeakerBio    = true;
+
+        if (!empty($this->taintedData['speaker_info'])) {
+            $validSpeakerInfo = $this->validateSpeakerInfo();
         }
 
-        if (!empty($this->_taintedData['speaker_bio'])) {
-            $valid_speaker_bio = $this->validateSpeakerBio();
+        if (!empty($this->taintedData['speaker_bio'])) {
+            $ValidSpeakerBio = $this->validateSpeakerBio();
```
Comment (mdwheele) [inline classes/Http/Form/SignupForm.php]: Lowercase the `V` in `$ValidSpeakerBio`

File: classes/Http/Form/SignupForm.php [hunk 5]
```diff
@@ -43,44 +43,44 @@ class SignupForm extends Form
...
-            $agree_coc;
+            $validEmail         &&
+            $validPasswords     &&
+            $validFirstName     &&
+            $validLastName      &&
+            $validCompany       &&
+            $validTwitter       &&
+            $validUrl           &&
+            $validSpeakerInfo   &&
+            $ValidSpeakerBio    &&
```
Comment (mdwheele) [inline classes/Http/Form/SignupForm.php]: `$validSpeakerBio`

File: classes/Domain/Services/ResetEmailer.php (hunk 3 above)
Comment (localheinz) [reply classes/Domain/Services/ResetEmailer.php]: How about in a separate PR?

Comment (BackEndTea) [reply classes/Domain/Services/ResetEmailer.php]: The silex Application object has the ```requireHttps()``` function we could use, and the host can be retrieved from the config file ( i think). So no need to use the ```Request``` object.

**Comment by localheinz** [comment]
Thank you, @mdwheele!

File: classes/Domain/Services/ResetEmailer.php (hunk 3 above)
Comment (localheinz) [reply classes/Domain/Services/ResetEmailer.php]: Sounds like a good plan to me, @BackEndTea!

**Pull request merged**